dreamers-app/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "react": "^16.1.0",
     "react-bootstrap": "^0.31.5",
+    "react-burger-menu": "^2.1.11",
```

Comment:
is this module still being used?  if not we need to remove it, i.e. uninstall

Reply:
i didnt know if we would come back to it. i think keeshas tutorial is from that module? could be wrong. we shall see!

Reply:
I don't think we should use it.. even after ejecting it's kind of a nightmare